### PR TITLE
Docstring [issue #5]

### PR DESCRIPTION
**Pull request by nicolas-enjalbert**

### CODE REVIEW

File: sanssouci/lambda_calibration.py [hunk 1]
```diff
@@ -11,33 +11,40 @@
...
-       Annals of Statistics, 48(3), 1281-1303.
+    Parameters
+    ----------
+    X : array-like of shape (n,p)
+        numpy array of size [n,p], containing n observations of p variables
+        (hypotheses)
+    categ : array-like of shape (n,)
+        numpy array of size [n], containing n values in {0, 1}, each of them
+        specifying the column indices of the first and the second sample.
+    B : int, default=100
```
Comment (bthirion) [inline sanssouci/lambda_calibration.py:26]:
```suggestion
    B : int,
```

File: sanssouci/lambda_calibration.py [hunk 2]
```diff
@@ -80,25 +87,32 @@ def get_perm_p(X, categ, B=100, row_test_fun=stats.ttest_ind):
...
-    Returns:
-     - A numpy array of of size [B]  containing the pivotal statitics,
-       whose j-th entry corresponds to \psi(g_j.X) with notation of
-       the AoS 2020 paper cited below (section 4.5)
-
-     Reference:
-     - Blanchard, G., Neuvial, P., & Roquain, E. (2020). Post hoc confidence
-       bounds on false positives using reference families.
-       Annals of Statistics, 48(3), 1281-1303.
+    r"""Get pivotal statistic
```
Comment (bthirion) [inline sanssouci/lambda_calibration.py:90]:
```suggestion
    """Get pivotal statistic
```

File: sanssouci/lambda_calibration.py [hunk 3]
```diff
@@ -80,25 +87,32 @@ def get_perm_p(X, categ, B=100, row_test_fun=stats.ttest_ind):
...
+
+    Parameters
+    ----------
+    p0 :  array-like of shape (B, p)
+        A numpy array of size [B,p] of null p-values obtained from
+        B permutations for p hypotheses.
+    t_inv : function, default=t_inv_linear
```
Comment (bthirion) [inline sanssouci/lambda_calibration.py:97]:
```suggestion
    t_inv : function
```
(the default comes at the end)

File: sanssouci/lambda_calibration.py [hunk 4]
```diff
@@ -80,25 +87,32 @@ def get_perm_p(X, categ, B=100, row_test_fun=stats.ttest_ind):
...
+        A function with the same I/O as t_inv_linear
+    K :  int, default=-1
```
Comment (bthirion) [inline sanssouci/lambda_calibration.py:99]: Same here

File: sanssouci/post_hoc_bounds.py [hunk 5]
```diff
@@ -72,18 +92,25 @@ def curve_max_fp(p_values, thr):
...
-               sorted non-decreasingly
-    * Returns:
-        - A vector of size p giving an joint upper confidence bound on
-          the number of false discoveries among the $k$ most significant
-          items for all k in \{1,\ldots,m\}
-    * Reference:
-        - Blanchard, G., Neuvial, P., & Roquain, E. (2020).
-        Post hoc confidence bounds on false positives using reference families.
+    Parameters
+    ----------
```
Comment (bthirion) [inline sanssouci/post_hoc_bounds.py:96]: I think that you should leave a blank after.

**Review by bthirion — COMMENTED** [review]
Great imporvements, thx !

File: sanssouci/post_hoc_bounds.py (hunk 5 above)
Comment (pneuvial) [reply sanssouci/post_hoc_bounds.py:96]: You mean a blank *line* after the '----------'?

Comment (bthirion) [reply sanssouci/post_hoc_bounds.py:96]: Exactly.

Comment (pneuvial) [reply sanssouci/post_hoc_bounds.py:96]: OK thanks. @nicolas-enjalbert  will take care of this when he's back (ie in April).

**Comment by pneuvial** [comment]
Thanks Nicolas!

**Pull request merged**